crates/rpc/types/src/types.rs

```diff
@@ -313,29 +313,26 @@ pub struct RpcDepositEntry {
     /// The outpoint that this deposit entry references.
     output: BitcoinOutPoint,
 
-    /// List of notary operators, by their indexes.
-    // TODO convert this to a windowed bitmap or something
-    notary_operators: Vec<OperatorIdx>,
+    /// Historical set of operators that formed the N/N multisig for this deposit.
```

Comment:
```suggestion
    /// The set of operators that formed the N/N multisig for this deposit.
```
I know it's the language used in ASM, but I don't really like it.